dataland-frontend/src/components/forms/parts/elements/derived/NaceCodeSelector.vue

```diff
@@ -75,6 +75,10 @@ export default defineComponent({
       type: Boolean,
       default: false,
     },
+    valueTree: {
```

Comment:
Why these changes?

Reply:
@MarcinStanczyk7

Reply:
as above:
"So that the NaceCode component can be used with other data than just Nace Codes. However, since we are not using NaceCodeComponent in DataRequest page but a Multiselect. There is no need to extend the NaceCodes component with this capability, so I will remove the code until it is needed."

Reply:
ok